lib/wx/configure.ac

```diff
@@ -40,24 +42,22 @@ AC_DEFUN([WX_MSG_ERROR],
 ])
 
 
-if test -z "$ERL_TOP" || test ! -d $ERL_TOP ; then
+AS_IF(
+ [test -z "$ERL_TOP" || test ! -d $ERL_TOP],
+ [
   AC_MSG_ERROR([ERL_TOP is not set])
-else
+ ],
+ [
   erl_top=${ERL_TOP}
-  AC_CONFIG_AUX_DIRS($erl_top/erts/autoconf)
   WX_BUILDING_INSIDE_ERLSRC=true
```

Comment:
This can be removed and all rest usages of it.

Reply:
Removed the standalone build leftovers